bin/main.rb

```diff
@@ -0,0 +1 @@
+puts "Hello World!"
```

Comment:
- [ ] You have to make the file executable, take some minutes to read[ this article](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) and make the file executable.